boards/arm/stm32h7/nucleo-h743zi/scripts/Make.defs

```diff
@@ -22,7 +22,15 @@ include $(TOPDIR)/.config
 include $(TOPDIR)/tools/Config.mk
 include $(TOPDIR)/arch/arm/src/armv7-m/Toolchain.defs
 
-LDSCRIPT = flash.ld
+ifeq ($(CONFIG_APP_FORMAT_MCUBOOT),y)
+ifeq ($(CONFIG_MCUBOOT_BOOTLOADER),y)
+    LDSCRIPT = flash-mcuboot-loader.ld
+else
+    LDSCRIPT = flash-mcuboot-app.ld
```

Comment:
Either correct this name to `flash-mcu-app.ld` or rename a linker script

Reply:
Done